coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelDataTypeToAvroType.java

```diff
@@ -110,9 +111,13 @@ private static Schema basicSqlTypeToAvroType(BasicSqlType relDataType) {
       case NULL:
         return Schema.create(Schema.Type.NULL);
       case TIMESTAMP:
-        Schema schema = Schema.create(Schema.Type.LONG);
+        schema = Schema.create(Schema.Type.LONG);
         schema.addProp("logicalType", "timestamp-millis");
         return schema;
+      case DATE:
+        schema = Schema.create(Schema.Type.INT);
```

Comment:
Since there's no `Date` type schema in avro https://github.com/apache/avro/blob/branch-1.10/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L125, how did we choose 'int' here? 

What is the default spark derived schema for 'Date' type field?

Reply:
https://github.com/apache/spark/blob/master/connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroDeserializer.scala#L145
Spark regards this avro schema as `date` type.

Reply:
so spark derived `date` type is cast-able to coral derived 'int' type?

Reply:
No, Spark AvroDeserializer would return `date` type directly for this case, and `spark.table.printSchema` also returns `date` type.